client/src/components/login/google-button/GoogleButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { GoogleLogin } from 'react-google-login';
+
+const GoogleButton = () => {
+  const responseGoogle = (response) => {
+    // TODO: 'post' call to Server with the id_token from response
+    console.log(response); // Placeholder till server code is integrated.
+  };
+
+  return (
+    <GoogleLogin
+      clientId={process.env.REACT_APP_GOOGLE_CLIENT_ID}
+      buttonText="Login with Google"
```

Comment:
What casing are we following?
This looks like title case, which is good.

Reply:
We are following PascalCase for react component throughout the project.